packages/webpack-cli/lib/groups/runHelp.js

```diff
@@ -112,10 +112,10 @@ const outputHelp = (args) => {
 
         const title = bold('⬡                     ') + underline('webpack') + bold('                     ⬡');
```

Comment:
Convert to tabs as well
```suggestion
        const title = bold('⬡\t\t\t\t\t') + underline('webpack') + bold('\t\t\t\t\t⬡');
```

Reply:
some issues with using `tabs` here. Don't know why but indentation is not equal on both sides if using tabs.